### ISSUE TITLE
Javascript

### ISSUE THREAD

**Issue by ir-fuel** [hanskellner/Fusion360Image2Surface#1]
Javascript is no longer supported in Fusion 360 :/

**Comment by hanskellner** [comment]
Yes, JavaScript development (editing/debugging) is no longer supported within Fusion 360.  But existing JavaScript add-ins can still be installed and run.

Take a look at this Screencast video which shows how to install the script:

[Load an existing Javascript script in Fusion](https://knowledge.autodesk.com/community/screencast/b5f14cfe-e117-496d-a767-0ed801ee9b48)
